lib/modules/manager/renovate-config-presets/index.ts

```diff
@@ -0,0 +1,20 @@
+import { configFileNames } from '../../../config/app-strings';
+import { GiteaTagsDatasource } from '../../datasource/gitea-tags';
+import { GithubTagsDatasource } from '../../datasource/github-tags';
+import { GitlabTagsDatasource } from '../../datasource/gitlab-tags';
+
+export { extractPackageFile } from './extract';
+
+export const url = '../../../config-presets.md';
```

Comment:
I'm not sure this makes sense. 🤔

Reply:
I tested the docs locally with `mkdocs serve` and confirmed it worked!